components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/ApplicationAuthenticator.java

```diff
@@ -40,7 +40,7 @@
      * @param request
      * @return boolean
      */
-    public boolean canHandle(HttpServletRequest request);
```

Comment:
what is the reason for this change?

Reply:
since this is an interface, defining it as public is redundant.